### ISSUE TITLE
Inquiry about the detail of collecting image-text pair from PDF

### ISSUE THREAD

**Issue by XiejiLi** [lxirich/MM-Retinal#3]
Dear authors, thank you for sharing your code and dataset collection workflow to us. What a wonderful work in 2024!

## Collect text data
When I read your paper and compare code with appendix detail. I ran your code in `Dataset_Collection/book1-2/step1_export_pdf_into_docx.py` with my adobe client id and client_secret, i only got docx file from adobe host, and no any image file received. In this step, I got the textbook information.
## Collect image data 
Here I thought we can get image data from adobe-api too, but we didn't get any image file from the host. According the paper **appendix** _**Step 1: Image-Text Pair Collection**._ paragraph, we need to capture image and cation in one screenshot. Is that a manual operation? Or can we make use of adobe-api? Do you mind to explain more on this part?
Thanks for your nice open source repos again,
Cheers.

**Comment by lxirich** [comment]
Thanks for your attention! 

For text collection, `Dataset_Collection/book1-2/step1_export_pdf_into_docx.py` is only used for the extraction of texts.

For image collection, we employed two different methods:

book1 and book2:
1. We first manually extracted image-text pairs from the books and named them accordingly (as shown in step1 of Figure 1(a) in our paper).
2. Then, we used the "Export PDF-Image" function in Adobe Acrobat software to export the images.
3. We utilized the Dataset_Collection/book1-2/step2_match.py script to match the corresponding images and text. 
This approach helps to avoid interference from irrelevant information in the diagram books and reduces the amount of manual proofreading later.
	
book3 and book4:
1. There was no need for manual screenshotting. We directly used the "Export PDF-Image" function in Adobe Acrobat software to export all the images from the entire book.
2. After that, we executed the five scripts in the Dataset_Collection/book3-4 folder sequentially. 
Since there might be irrelevant content in the books, this method may require more manual proofreading.

We hope this clarifies our process.

**Comment by lxirich** [comment]
Thanks for your attention! 

`Dataset_Collection/book1-2/step1_export_pdf_into_docx.py` is used  to convert the pdf to docx for image text matching in `Dataset_Collection/book1-2/step2_match.py`.

For image collection, we employed two different methods:

book1 and book2:
1. We manually captured image-text pairs from the diagram books (as shown in step1 of Figure 1(a) in our paper) and used the Adobe Acrobat to export the images.
2. We utilized the Dataset_Collection/book1-2/step2_match.py to match the corresponding images and text.
This approach helps to avoid the irrelevant information in the diagram books and reduces the manual proofreading work.

book3 and book4:
1. There is no need for manual screenshotting. We directly used the Adobe Acrobat to export all the images from the entire book and use OCR api to convert the books to docx.
2. We executed the five scripts in the Dataset_Collection/book3-4 folder sequentially.
Because this method directly OCR the entire book instead of captured image-text pairs, it requires more manual proofreading to remove the irrelevant content.

We hope this clarifies our process.

**Comment by lxirich** [comment]
Thanks for your attention!

`Dataset_Collection/book1-2/step1_export_pdf_into_docx.py` is used to convert the pdf to docx for image text matching in `Dataset_Collection/book1-2/step2_match.py`.

For image collection, we employed two different methods:

book1 and book2:

1. We manually captured image-text pairs from the diagram books (as shown in step1 of Figure 1(a) in our paper) and used the Adobe to export the images.
2. We utilized the Dataset_Collection/book1-2/step2_match.py to match the corresponding images and text.
This approach helps to avoid the irrelevant information in the diagram books and reduces the manual proofreading work.

book3 and book4:

1. There is no need for manual screenshotting. We directly used the Adobe to export all the images from the entire book and use OCR api to convert the books to docx.
2. We executed the five scripts in the Dataset_Collection/book3-4 folder sequentially.
Because this method directly OCR the entire book instead of captured image-text pairs, it requires more manual proofreading to remove the irrelevant content.

We hope this clarifies our process.
